tutorials/nginx-ingress-gke/index.md

```diff
@@ -0,0 +1,353 @@
+generate a cost estimate based on your projected usage.
+
+# Before You Begin
+
+1. You'll need a Google Cloud Platform (GCP) project. Select or create a GCP project.  
+   [GO TO THE PROJECTS PAGE](https://console.cloud.google.com/project)
+   
+1. Enable billing for your project.  
+	[ENABLE BILLING](https://support.google.com/cloud/answer/6293499#enable-billing)
+
+1. Enable the Google Kubernetes Engine (GKE) API.  
+	[ENABLE APIs](https://console.cloud.google.com/flows/enableapi?apiid=container,cloudresourcemanager.googleapis.com)
+
+# Set up your environment
+
+In this section you configure the infrastructure and identities required to
+complete the tutorial.
+
+## Start a Cloud Shell instance and create a Kubernetes Engine cluster
+
+1. You'll run all the terminal commands in this tutorial from Cloud Shell.  
+	[OPEN CLOUD SHELL](https://console.cloud.google.com/?cloudshell=true)
+
+1. Create a Kubernetes Engine cluster to deploy Nginx and the sample
+application with the following commands:  
+  
+        gcloud config set compute/zone us-central1-f  
+        gcloud container clusters create nginx-tutorial --machine-type=n1-standard-1 --num-nodes=2
+        
+1. From the Cloud Shell, clone the following repo which contains all the
+files for this demo:
+    
+    	git clone https://github.com/ameer00/nginx-ingress-gke 
+    	cd nginx-ingress-gke
+
+# Deploy an application in GKE
+
+You will be deploying a simple web based application from the Google Cloud
+Repository, courtesy of
+[Kubernetes Up and Running repo](https://github.com/kubernetes-up-and-running/kuard).
+ You will use this application as the backend for our Ingress.
+
+From the Cloud Shell:  
+  
+	kubectl apply -f kuard-app.yaml
+
+	service "kuard" created
+	deployment "kuard" created
+
+Verify that your deployment is running 3 replica pods and the service is exposed.
+
+	kubectl get deployments kuard
+
+	NAME      DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+	kuard     3         3         3            3           1m
+
+	kubectl get pods
+
+	NAME                     READY     STATUS    RESTARTS   AGE
+	kuard-2740446302-03p3b   1/1       Running   0          1m
+	kuard-2740446302-6k65c   1/1       Running   0          1m
+	kuard-2740446302-wbj3g   1/1       Running   0          1m
+
+	kubectl get service kuard
+
+	NAME      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)   AGE
+	kuard     ClusterIP   10.7.253.136   <none>        80/TCP    8s
+
+# Deploy a default backend for Ingress
+
+Next, you will deploy a default backend for the nginx ingress.  The default
+backend is a service which handles all url paths and hosts the nginx controller
+doesn't understand (i.e., all the requests that are not mapped with an Ingress
+Resource). The default backend exposes two URLs:  
+
+-  `/healthz` that returns 200
+-  `/` that returns 404
+
+In Cloud Shell, deploy the default backend deployment and service.
```

Comment:
"From Cloud Shell, deploy the default backend Deployment and Service by running the following command:"